hphp/runtime/vm/jit/smashable-instr-arm.h

```diff
@@ -77,6 +77,22 @@ constexpr size_t kSmashJccTargetOff = 12;
 
 ///////////////////////////////////////////////////////////////////////////////
 
+template<typename T>
+  static void patchInstr(TCA inst, T target) {
+  *reinterpret_cast<T*>(inst) = target;
+  auto const end = reinterpret_cast<char*>(inst + sizeof(T));
+  auto const begin = end - sizeof(T);
+  __builtin___clear_cache(begin, end);
+}
```

Comment:
I think it's be nicer to have differently-named versions of this for `uint64_t` and `uint32_t`, since we really want to be clear which one we're using at every point.